src/traces/scatterpolargl/defaults.js

```diff
@@ -35,6 +35,7 @@ module.exports = function supplyDefaults(traceIn, traceOut, defaultColor, layout
     coerce('mode', len < PTS_LINESONLY ? 'lines+markers' : 'lines');
     coerce('text');
     coerce('hovertext');
+    coerce('hovertemplate');
```

Comment:
**N.B.** `hovertemplate` should not get coerced for `hoveron: 'fills'`, just like for `scatter` traces:

https://github.com/plotly/plotly.js/blob/a78600a5689696b63e051454a6976c242698bc18/src/traces/scatter/defaults.js#L78